### PR TITLE
[Doc][Kommander] Kommander 1.2.1 Release Notes for Patch.

### PR DESCRIPTION
**Pull request by russurquhart1**
## Jira Ticket
Before creating this pull request, make sure you have a separate ticket for the docs team to track their work. If you need assistance, ask in the #documentation Slack channel.

## Description of changes being made
Release Notes for Kommander 1.2.1 Patch

## Checklist
- [ ] Change all affected versions, if applicable (e.g. 1.13, 2.0, 2.1).
- [ ] Test all commands and procedures, if applicable.
- [ ] Create your PR against `main`. 
- [ ] Update all links if you are moving a page.
- [ ] Add release date to Release Notes page in the following format: <Package> was released on <Day>, <Month> <Year> Example: `Mesosphere® DC/OS™ 2.1.0 was released on 9, June 2020`

See the [contribution guidelines](https://wiki.d2iq.com/display/ENG/Contributing+to+Docs) for more information.

### CODE REVIEW

File: pages/dkp/kommander/1.2/release-notes/1.2.1/index.md [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+# Supported Versions
+| Kubernetes Support | Version |
+| ------------------ | ------- |
+| **Minimum**        | 1.17.0  |
+| **Maximum**        | 1.19.x  |
+| **Default**        | 1.18.10  |
+
+# Breaking changes
+
+In Kommander 1.3, all new projects were created with a default NetworkPolicy that only allowed traffic originating within your Projects namespace. This policy was also created in all existing projects that did not already have a NetworkPolicy when upgrading to Kommander 1.3.
```
Comment (gracedo) [inline pages/dkp/kommander/1.2/release-notes/1.2.1/index.md:31]:
```suggestion
```

This is a 1.3 specific feature so we can remove all this

File: pages/dkp/kommander/1.2/release-notes/1.2.1/index.md [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+
+The result of this default policy was that Pods could not talk to Pods outside of that namespace. In some cases this was not a desired default state. As of Kommander 1.3.1, no default NetworkPolicy is created in new (or existing projects) and thus no traffic blocking is done by default.
```
Comment (gracedo) [inline pages/dkp/kommander/1.2/release-notes/1.2.1/index.md:33]:
```suggestion
```

File: pages/dkp/kommander/1.2/release-notes/1.2.1/index.md [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+| Kubernetes Support | Version |
+| ------------------ | ------- |
+| **Minimum**        | 1.17.0  |
+| **Maximum**        | 1.19.x  |
+| **Default**        | 1.18.10  |
+
+# Breaking changes
+
+In Kommander 1.3, all new projects were created with a default NetworkPolicy that only allowed traffic originating within your Projects namespace. This policy was also created in all existing projects that did not already have a NetworkPolicy when upgrading to Kommander 1.3.
+
```
Comment (gracedo) [inline pages/dkp/kommander/1.2/release-notes/1.2.1/index.md:32]:
```suggestion
```

File: pages/dkp/kommander/1.2/release-notes/1.2.1/index.md [hunk 4]
```diff
@@ -0,0 +1,68 @@
...
+The result of this default policy was that Pods could not talk to Pods outside of that namespace. In some cases this was not a desired default state. As of Kommander 1.3.1, no default NetworkPolicy is created in new (or existing projects) and thus no traffic blocking is done by default.
+
```
Comment (gracedo) [inline pages/dkp/kommander/1.2/release-notes/1.2.1/index.md:34]:
```suggestion
```

File: pages/dkp/kommander/1.2/release-notes/1.2.1/index.md [hunk 5]
```diff
@@ -0,0 +1,68 @@
...
+If you are upgrading from Kommander 1.3 and already made the necessary changes to address this issue (for example, by adding a NetworkPolicy with the desired traffic rules), then upgrading to Kommander 1.3.1 will not impact those existing policies.
```
Comment (gracedo) [inline pages/dkp/kommander/1.2/release-notes/1.2.1/index.md:35]:
```suggestion
```

File: pages/dkp/kommander/1.2/release-notes/1.2.1/index.md [hunk 6]
```diff
@@ -0,0 +1,68 @@
...
+# New features
+
+- Add license delete mutation.
```
Comment (gracedo) [inline pages/dkp/kommander/1.2/release-notes/1.2.1/index.md:38]:
```suggestion
```

File: pages/dkp/kommander/1.2/release-notes/1.2.1/index.md [hunk 7]
```diff
@@ -0,0 +1,68 @@
...
+- Replace license table with single license detail view.
```
Comment (gracedo) [inline pages/dkp/kommander/1.2/release-notes/1.2.1/index.md:39]:
```suggestion
```

File: pages/dkp/kommander/1.2/release-notes/1.2.1/index.md [hunk 8]
```diff
@@ -0,0 +1,68 @@
...
+- Allow workspace namespace to be configurable.
```
Comment (gracedo) [inline pages/dkp/kommander/1.2/release-notes/1.2.1/index.md:40]:
```suggestion
```

File: pages/dkp/kommander/1.2/release-notes/1.2.1/index.md [hunk 9]
```diff
@@ -0,0 +1,68 @@
...
+
+# Breaking changes
+
+In Kommander 1.3, all new projects were created with a default NetworkPolicy that only allowed traffic originating within your Projects namespace. This policy was also created in all existing projects that did not already have a NetworkPolicy when upgrading to Kommander 1.3.
+
+The result of this default policy was that Pods could not talk to Pods outside of that namespace. In some cases this was not a desired default state. As of Kommander 1.3.1, no default NetworkPolicy is created in new (or existing projects) and thus no traffic blocking is done by default.
+
+If you are upgrading from Kommander 1.3 and already made the necessary changes to address this issue (for example, by adding a NetworkPolicy with the desired traffic rules), then upgrading to Kommander 1.3.1 will not impact those existing policies.
+# New features
+
```
Comment (gracedo) [inline pages/dkp/kommander/1.2/release-notes/1.2.1/index.md:37]:
```suggestion
```

File: pages/dkp/kommander/1.2/release-notes/1.2.1/index.md [hunk 10]
```diff
@@ -0,0 +1,68 @@
...
+- Add license delete mutation.
+- Replace license table with single license detail view.
+- Allow workspace namespace to be configurable.
+
+# Bug fixes
+
+- Ensure pre-delete hook jobs are cleaned up.
+- Ensure kubectl deletes do not fail if resource already deleted.
+- Fix empty non-Konvoy cluster Platform Services tab.
```
Comment (gracedo) [inline pages/dkp/kommander/1.2/release-notes/1.2.1/index.md:46]:
```suggestion
- Fix empty non-Konvoy cluster Platform Services tab.
```
```suggestion
- Bump kubecost to fix an issue that occasionally caused pods to fail to deploy.
```

File: pages/dkp/kommander/1.2/release-notes/1.2.1/index.md [hunk 11]
```diff
@@ -0,0 +1,68 @@
...
+- Disable all platform services when Foundational Components are disabled in the UI.
```
Comment (gracedo) [inline pages/dkp/kommander/1.2/release-notes/1.2.1/index.md:47]:
```suggestion
- Fix bug in Kommander UI where LDAP Root CA is malformed when saved.
```

File: pages/dkp/kommander/1.2/release-notes/1.2.1/index.md [hunk 12]
```diff
@@ -0,0 +1,68 @@
...
+- All federated platform services are now successfully removed when the kubeaddons controller is disabled.
```
Comment (gracedo) [inline pages/dkp/kommander/1.2/release-notes/1.2.1/index.md:48]:
```suggestion
```

File: pages/dkp/kommander/1.2/release-notes/1.2.1/index.md [hunk 13]
```diff
@@ -0,0 +1,68 @@
...
+- Kubecost: Ensure kubectl deletes do not fail if resource is already deleted.
+- When self attaching a kommander cluster, the dex-k8s-authenticator configmap is no longer updated as this causes invalid links on the generate token page.
```
Comment (gracedo) [inline pages/dkp/kommander/1.2/release-notes/1.2.1/index.md:50]:
```suggestion
```

File: pages/dkp/kommander/1.2/release-notes/1.2.1/index.md [hunk 14]
```diff
@@ -0,0 +1,68 @@
...
+- Allow workspace namespace to be configurable.
+
+# Bug fixes
+
+- Ensure pre-delete hook jobs are cleaned up.
+- Ensure kubectl deletes do not fail if resource already deleted.
+- Fix empty non-Konvoy cluster Platform Services tab.
+- Disable all platform services when Foundational Components are disabled in the UI.
+- All federated platform services are now successfully removed when the kubeaddons controller is disabled.
+- Kubecost: Ensure kubectl deletes do not fail if resource is already deleted.
```
Comment (gracedo) [inline pages/dkp/kommander/1.2/release-notes/1.2.1/index.md:49]:
```suggestion
```

File: pages/dkp/kommander/1.2/release-notes/1.2.1/index.md [hunk 15]
```diff
@@ -0,0 +1,68 @@
...
+- When self attaching a kommander cluster, the dex-k8s-authenticator configmap is no longer updated as this causes invalid links on the generate token page.
+- Fixes bug in Kommander UI where LDAP Root CA is not correct when saved.
```
Comment (gracedo) [inline pages/dkp/kommander/1.2/release-notes/1.2.1/index.md:51]:
```suggestion
```

File: pages/dkp/kommander/1.2/release-notes/1.2.1/index.md [hunk 16]
```diff
@@ -0,0 +1,68 @@
...
+- Updates UI to only ship with needed dependencies.
```
Comment (gracedo) [inline pages/dkp/kommander/1.2/release-notes/1.2.1/index.md:52]:
```suggestion
```

File: pages/dkp/kommander/1.2/release-notes/1.2.1/index.md [hunk 17]
```diff
@@ -0,0 +1,68 @@
...
+
+## Component versions
+
+- Addon: 1.3.1-5
```
Comment (gracedo) [inline pages/dkp/kommander/1.2/release-notes/1.2.1/index.md:56]:
```suggestion
- Addon: 1.2.1-2
```

File: pages/dkp/kommander/1.2/release-notes/1.2.1/index.md [hunk 18]
```diff
@@ -0,0 +1,68 @@
...
+- Chart: 0.15.3
```
Comment (gracedo) [inline pages/dkp/kommander/1.2/release-notes/1.2.1/index.md:57]:
```suggestion
- Chart: 0.12.19
```

File: pages/dkp/kommander/1.2/release-notes/1.2.1/index.md [hunk 19]
```diff
@@ -0,0 +1,68 @@
...
+- kommander-federation (yakcl): 0.8.8
```
Comment (gracedo) [inline pages/dkp/kommander/1.2/release-notes/1.2.1/index.md:58]:
```suggestion
- kommander-federation (yakcl): 0.6.10
```

File: pages/dkp/kommander/1.2/release-notes/1.2.1/index.md [hunk 20]
```diff
@@ -0,0 +1,68 @@
...
+- kommander-licensing (yakcl): 0.8.8
```
Comment (gracedo) [inline pages/dkp/kommander/1.2/release-notes/1.2.1/index.md:59]:
```suggestion
- kommander-licensing (yakcl): 0.6.10
```

File: pages/dkp/kommander/1.2/release-notes/1.2.1/index.md [hunk 21]
```diff
@@ -0,0 +1,68 @@
...
+- UI: 6.91.1
```
Comment (gracedo) [inline pages/dkp/kommander/1.2/release-notes/1.2.1/index.md:60]:
```suggestion
- UI: 6.55.2
```

File: pages/dkp/kommander/1.2/release-notes/1.2.1/index.md [hunk 22]
```diff
@@ -0,0 +1,54 @@
...
+# New features
+
+- Replace license table with single license detail view.
+
+# Bug fixes
+
+- Ensure pre-delete hook jobs are cleaned up.
+- Ensure kubectl deletes do not fail if resource already deleted.
+- Bump kubecost to fix an issue that occasionally caused pods to fail to deploy
+- Fix bug in Kommander UI where LDAP Root CA is malformed when saved.
```
Comment (cneth) [inline pages/dkp/kommander/1.2/release-notes/1.2.1/index.md:38]:
```suggestion
- Fix bug in Kommander UI where LDAP Root CA is malformed when saved. (COPS-6884)
```

File: pages/dkp/kommander/1.2/version-policy/index.md [hunk 23]
```diff
@@ -0,0 +1,76 @@
...
+
+When the 1.4 version releases, support continues for 1.3, and 1.2. Support for Kommander version 1.1.x expires. Users should upgrade their Kommander clusters with every new release to stay up-to-date with the latest features and bug fixes.
+
+You can read more about our official support policy in [D2iQ Support and Maintenance Terms](https://d2iq.com/legal/support-terms).
+
+## Supported Kubernetes versions
+
+Each Kommander release supports a range of Kubernetes versions. The [Release Notes](../release-notes) describe these versions.
+
+For example, Kommander 1.3.0 supports:
```
Comment (cneth) [inline pages/dkp/kommander/1.2/version-policy/index.md:21]:
```suggestion
For example, Kommander 1.2.0 supports:
```

File: pages/dkp/kommander/1.2/version-policy/index.md [hunk 24]
```diff
@@ -0,0 +1,76 @@
...
+| Konvoy Version | Kommander Version |
+| -------------- | ----------------- |
+| 1.7.x | 1.3.x |
+| 1.6.x | 1.2.x |
+| 1.5.x | 1.1.x |
+
+Konvoy and Kommander release `MAJOR.MINOR` versions together and are compatible with each other for that version set. This means, Kommander 1.3, and all of its patch versions (e.g. 1.3.0, 1.3.1), can deploy successfully on any version of Konvoy 1.7. Mixing minor versions is prohibited and the following is not supported:
+
+- You cannot deploy Kommander 1.2.x on Konvoy 1.7.x
+- You cannot deploy Kommander 1.1.x on Konvoy 1.6.x
```
Comment (cneth) [inline pages/dkp/kommander/1.2/version-policy/index.md:42]: This mentions Kommander 1.3 but this is a 1.2 page.

```suggestion
Konvoy and Kommander release `MAJOR.MINOR` versions together and are compatible with each other for that version set. This means, Kommander 1.2, and all of its patch versions (e.g. 1.2.0, 1.2.1), can deploy successfully on any version of Konvoy 1.6. Mixing minor versions is prohibited and the following is not supported:

- You cannot deploy Kommander 1.2.x on Konvoy 1.7.x
- You cannot deploy Kommander 1.1.x on Konvoy 1.6.x
```

**Review by cneth — CHANGES_REQUESTED** [review]

File: pages/dkp/kommander/1.2/release-notes/1.2.1/index.md [hunk 25]
```diff
@@ -0,0 +1,54 @@
...
+
+# Release Summary
+Kommander provides a command center for all your cloud native management needs in public Information as a Service (IaaS), on-premises, and edge environments. Kommander provides a multi-tenant experience to create, secure, and configure Kubernetes clusters and cloud native workloads. Additionally, Kommander enables teams to unlock federated cost management across multiple clusters, whether they are a new Konvoy cluster or an existing 3rd party/DIY distribution installation.
+
+# Supported Versions
+| Kubernetes Support | Version |
+| ------------------ | ------- |
+| **Minimum**        | 1.17.0  |
+| **Maximum**        | 1.19.x  |
+| **Default**        | 1.18.10  |
```
Comment (cneth) [inline pages/dkp/kommander/1.2/release-notes/1.2.1/index.md:27]: This table confuses me...  Kommander doesn't install Kubernetes.  The 'Default values' here don't match what is in the Konvoy 1.6.2 rn.

**Review by cneth — APPROVED** [review]

**Pull request merged**